content/en/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -170,6 +170,33 @@ to access a replica via its ephemeral public IP, you must skip TLS verification.
 To allow etcd clustering, ports needed to communicate between etcd instances will be opened (for inside cluster communication).
 To make such deployment secure, communication between etcd instances is authorized using SSL.
 
+### API server identity
+
+{{< feature-state state="alpha"  for_k8s_version="v1.20" >}}
+
+The API Server Identity feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate API Server Identity by enabling
+the feature gate named `APIServerIdentity` when you start the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
+
+```shell
+kube-apiserver \
+--feature-gates=APIServerIdentity=true \
+ # …and other flags as usual
+```
+
+During bootstrap, each kube-apiserver assigns a unique ID to itself. The ID is
+in the format of `kube-apiserver-{UUID}`. Each kube-apiserver creates a
+[Lease](/docs/reference/generated/kubernetes-api/{{< param "version" >}}//#lease-v1-coordination-k8s-io)
+in the _kube-system_ {{< glossary_tooltip text="namespaces" term_id="namespace">}}.
+The Lease name is the unique ID for the kube-apiserver. The Lease contains a
+label `k8s.io/component=kube-apiserver`. Each kube-apiserver manages deleting
+identity leases for kube-apiservers that are gone.
+
+Enabling this feature is a prerequisite for using features that involve HA API
+server coodination (e.g. the StorageVersion API feature).
```

Comment:
nits (we avoid Latin abbreviations; typo fix)
```suggestion
Enabling this feature is a prerequisite for using features that involve HA API
server coordination (for example, the StorageVersion API feature).
```

Is StorageVersion a `kind` or the name of a feature gate? I'm not sure readers can work that out.

Reply:
it's the feature gate. Updated